### PR TITLE
Js stealth mode example

### PR DESCRIPTION
**Pull request by TinyGambe**
*Does not work*. Blocked by `PageMonitor` error

### CODE REVIEW

File: javascript-sdk/examples/stealth-mode/stealth-mode.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    const acceptLanguage = ACCEPT_LANGUAGES[Math.floor(Math.random() * ACCEPT_LANGUAGES.length)];
+    const proxy= PROXIES.length > 0 ? PROXIES[Math.floor(Math.random() * PROXIES.length)] : null;
+
+    const browser = await chromium.launch({
+        headless: false,
+        args: BROWSER_ARGS,
+        ignoreDefaultArgs: BROWSER_IGNORED_ARGS,
+    });
+
+    configure({ apiKey: 'Ngq7twL_y4-IHjIpvc5dSvQkyz82RQtd6fi2VnHldkJlE0VDN8S7sg' });
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/stealth-mode.js:64]: Hi Gambe! Is this the production API key? If so, you may need to delete it as it's exposed in a public repo.

File: javascript-sdk/examples/stealth-mode/stealth-mode.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        geolocation: location[1],
+        userAgent: userAgent,
+        permissions: ['notifications'],
+        viewport: {
+            width: 1920 + randomInt(-50, 50),
+            height: 1080 + randomInt(-50, 50),
+        },
+    });
+
+    const page = wrap(context.newPage());
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/stealth-mode.js:86]: I think what's causing the issue is that you're not awaiting for `newPage()` here.

File: javascript-sdk/examples/stealth-mode/stealth-mode.js (hunk 1 above)
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/stealth-mode.js:64]: We're planning to remove this line from examples.

**Review by frankfeng98 — CHANGES_REQUESTED** [review]
Thank you for the work! We just released a new version of JS SDK. You might need to update the package in order for script to work. In addition, you may need to delete the API key included in this script from our database.

File: javascript-sdk/examples/stealth-mode/stealth-mode.js (hunk 1 above)
Comment (TinyGambe) [reply javascript-sdk/examples/stealth-mode/stealth-mode.js:64]: Yikes... Can I delete it from the dev portal or do I need to go to the database?

Comment (frankfeng98) [reply javascript-sdk/examples/stealth-mode/stealth-mode.js:64]: I think delete from the dev portal should be sufficient? 

cc: @taha-tf

File: javascript-sdk/examples/stealth-mode/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- randomize various HTTP headers browser sends to the server. This includes `User-Agent`, `Accept-Language`, `Referer`, etc. This helps with consecutive requests looking more like they are coming from different users.
+- randomize browser window size. This is important because some websites track the window size and if it's the same for all requests, it's a sign of a bot.
+- randomize timezone and geolocation. This is important because some websites track the timezone and geolocation and if it's the same for all requests, it's a sign of a bot.
+- (Optional) use a proxy server. You would need to get a Proxy configuration (host, username, password) separately from an external proxy provider (e.g. [NetNut](https://netnut.io), [BrightData](https://brightdata.com/) or similar)
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/installation/sdk-installation)
+  - If you already have SDK installed, make sure to update to the latest version: `npm install agentql --upgrade`
+- Save this javascript file locally as **stealth-mode.js**
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/README.md:16]: javascript -> Javascript

File: javascript-sdk/examples/stealth-mode/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+There are several techniques we use in this script to avoid detection:
+
+- randomize various HTTP headers browser sends to the server. This includes `User-Agent`, `Accept-Language`, `Referer`, etc. This helps with consecutive requests looking more like they are coming from different users.
+- randomize browser window size. This is important because some websites track the window size and if it's the same for all requests, it's a sign of a bot.
+- randomize timezone and geolocation. This is important because some websites track the timezone and geolocation and if it's the same for all requests, it's a sign of a bot.
+- (Optional) use a proxy server. You would need to get a Proxy configuration (host, username, password) separately from an external proxy provider (e.g. [NetNut](https://netnut.io), [BrightData](https://brightdata.com/) or similar)
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/installation/sdk-installation)
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/README.md:14]: Use this link:

https://docs.agentql.com/javascript-sdk/installation

File: javascript-sdk/examples/stealth-mode/stealth-mode.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    geolocation: location[1],
+    userAgent: userAgent,
+    permissions: ['notifications'],
+    viewport: {
+      width: 1920 + randomInt(-50, 50),
+      height: 1080 + randomInt(-50, 50),
+    },
+  });
+
+  const page = wrap(await context.newPage());
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/stealth-mode.js:81]: We just updated `wrap()` to be async, could you await it? Thank you!

**Review by frankfeng98 — CHANGES_REQUESTED** [review]
The script looks good overall! Just several nits to address

File: javascript-sdk/examples/stealth-mode/stealth-mode.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+  });
+
+  const page = await wrap(await context.newPage());
+  await page.goto('https://bot.sannysoft.com/', { referer });
+  await page.waitForTimeout(30000);
+
+  await browser.close();
+}
+
+main();
```
Comment (frankfeng98) [inline javascript-sdk/examples/stealth-mode/stealth-mode.js:88]: This should have `await`

**Review by frankfeng98 — APPROVED** [review]
LGTM! Please fix one more nit before merging

**Pull request merged**